readme.md

```diff
@@ -27,14 +25,24 @@ Once the app is up and running you can use it manually by visiting the launch pa
 |`iss`|The base URL of the FHIR server|
 |`patientId`|The ID of the patient in context|
 
-The FHIR server most properly comply with the SMART on FHIR specification and have a conformace statement with a `security` section that contains the authorization endpoints that the app needs to request a token from.  
+The FHIR server must properly comply with the SMART on FHIR specification and have a conformace statement with a `security` section that contains the authorization endpoints that the app needs to request a token from.  
 
 For example, 
 http://localhost:3050/launch?iss=http://launch.smarthealthit.org/v/r2/fhir/&patientId=Patient/ab2931ba-6c2d-4110-8028-b5a95cd9f5c7 is an example of a valid launch url that will successfully open the app.  The app will display the name of the patient if it successfully authenticates and retrieves the patient resource.
 
-## Connecting with other subprojects
```

Comment:
This should still be a `connecting with other subprojects` section, the only server that is absolutely needed to run this app is the CRD server.  The EHR, Request generator, and Keycloak are not necessary if you're running the SMART app with a different EHR.